rfcs/models/identity/identity-update-policy.md

```diff
@@ -0,0 +1,69 @@
+
+The memberships in a blockchain network are not guaranteed to be static. Often existing participant members of a network leave while new participants join. As a result, the `verificationMethod` and `networkParticipants` of the *Network DID* must also be updated to capture the updated network structure.
+
+Based on the different governance policies of different networks, they might want to enforce different policies that dictate which combination of exisiting parcitipants of the network are allowed to introduce a new member or expel existing members. These policicies can be encoded into the `updatePolicy` property of the *BlockchainNetworkMultiSig* `verificationMethod` in the *Network DID*.
+
+`updatePolicy` defines a combination of participant units that can sign and authorize a Network DID updation request. This `updatePolicy` has the condition format as specified in [`verifiablecondition2021`](https://w3c.github.io/did-spec-registries/#verifiablecondition2021) verification method. It can be used to combine different verification methods together to create complex conditional expressions such as logical operations, thresholds, weighted thresholds, relationships and a delegation to external verification methods.
+
+
+Example of `updatePolicy`. 
+```json
+{
+  "id": "did:<iin_name>:<network_name>#updatepolicy",
+  "controller": "did:<iin_name>:<network_name>",
+  "type": "VerifiableCondition2021",
+  "conditionAnd": [{
+      "id": "did:<iin_name>:<network_name>#updatepolicy-1",
+      "controller": "did:<iin_name>:<network_name>",
+      "type": "VerifiableCondition2021",
+      "conditionOr": ["did:<iin_name>:<network_participant_3>#key1",
+        "did:<iin_name>:<network_participant_2>#key3"
+      ]
+    },
+    "did:<iin_name>:<network_participant_1>#key1"
+  ]
+}
+```
+
+## Network DID updates
+
+Any request to update the *Network DID* must be authenticated based on different policies for the different scenarios of (i) participants leaving the network, (ii) new participants joining the network, and (iii) other updates to the Network DID.
+
+### Participants leaving the network
+
+There can be two scenarios when an existing participant unit of a network leaves: (a) The participant itself willingly leaves the network. (b) Other participants of the network decide to expel that participant. Based on these two scenarios, any *Network DID* update request removing one participant from the `networkParticipants` property is authenticated as per the following conditions:
+
+A. The *Network DID* update request is attested by the participant who is leaving the network, that is the participant whose DID is being excluded from the  `networkParticipants` property. This indicates voluntary exit from the network.
+
+B. If condition A is not satisfied, and there exists an `updatePolicy` property of the *BlockchainNetworkMultiSig* `verificationMethod` in the *Network DID*, then the DID update request must be attested satisfying the `updatePolicy`.
```

Comment:
It'd be useful to add that this policy contains, among other things, a member list that excludes one or more earlier participants.